Exiled.API/Features/Roles/Scp3114Role.cs

```diff
@@ -217,6 +217,15 @@ public void ResetIdentity()
             Identity.CurIdentity.Reset();
             Identity.ServerResendIdentity();
         }
+        
+        /// <summary>
+        /// Gets or sets wasDisguised boolean.
+        /// </summary>
+        public bool WasDisguised
+        {
+            get => Identity._wasDisguised;
+            set => Identity._wasDisguised = value;
```

Comment:
Can you make sure than setter update with client (i actually thinks than it's don't and than it's will imediatelly be overided)